pkg/virtctl/expose/OWNERS

```diff
@@ -0,0 +1,6 @@
+reviewers:
+  - sig-network-reviewers
+approvers:
+  - sig-network-approvers
+labels:
+  - sig/network
```

Comment:
I'm not supper happy about this one (I think we should not support this at all), but it does make sense and reflects reality.